estimator/util.py

```diff
@@ -4,16 +4,46 @@
 from dataclasses import dataclass
 from typing import Any, Callable, NamedTuple
 
-from sage.all import ceil, floor, log, oo
+from sage.all import ceil, floor, log, oo, cached_function
+from scipy.special import zeta
```

Comment:
Why scipy and not Sage?

Reply:
scipy zeta function returns +inf on zeta(1), sage zeta function returns an unsigned Infinity, which can't be converted to RR.

Reply:
Still, I think we should use Sage's functions, it reads easier?

Reply:
It definitely does. But it straight up breaks everything. I can't reassign unsigned Infinity to +infinity (oo), they're derived from two completely different objects (Signed Infinity Ring v.s. Unsigned Infinity Ring).

Reply:
Kay, let's leave it as is but document why?